aws/client.go

```diff
@@ -0,0 +1,302 @@
+package aws
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"text/template"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	awsConfig "github.com/aws/aws-sdk-go-v2/config"
```

Comment:
```suggestion
	awsconfig "github.com/aws/aws-sdk-go-v2/config"
```

It's a package name, and like so, they are lowercase

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. 

https://go.dev/blog/package-names